content/en/platform/corda/5.0-alpha/deploying/installing-corda-cli.md

```diff
@@ -14,19 +14,19 @@ Corda CLI (command line interface) is a command line tool that supports various
 
 To obtain the Corda CLI installer:
 1. Download `platform-jars-Eagle.tar.gz` from the [R3 Customer Hub](https://r3.force.com/).
-2. Extract `corda-cli-downloader-Eagle.zip` from `net\corda\cli\deployment\corda-cli-installer\5.0.0.0-Eagle`.
+2. Extract `corda-cli-installer-5.0.0.0-Eagle.zip` from `net\corda\cli\deployment\corda-cli-installer\5.0.0.0-Eagle`.
 
```

Comment:
Since this tool assumes that `java` is installed on a coputer, we should probably add a pre-requisite step for people to check that `java` is indeed installed and is of compatible version.

Reply:
Thanks @vkolomeyko. What version is required?

Reply:
It is definitely JRE 11, but which specific Java vendors we are actually supporting was a question that I do not  remember seeing an aswer for from product.

When I am running CLI locally, I am using "OpenJDK Runtime Environment AdoptOpenJDK (build 11.0.9+11)" and pretty happy with it.

Reply:
Added Azul JDK 11